src/Psl/Str/Byte/replace_ci.php

```diff
@@ -12,5 +14,5 @@
  */
 function replace_ci(string $haystack, string $needle, string $replacement): string
```

Comment:
ci confused me with `continuous integration` at first. No big deal though

Reply:
ci here ( and in another Str\ functions ), indicates `case insensative`, I think we need to clearify that in the documentation, but i can't see a better name for it ( replace_case_insensative? too long? )

Reply:
That is too long indeed... ci is ok - just something you need to know. It makes sense in the context.